source/braille.py

```diff
@@ -2499,7 +2503,18 @@ def _handlePendingUpdate(self):
 			scrollTo: Optional[TextInfoRegion] = None
 			self.mainBuffer.saveWindow()
 			for region in self._regionsPendingUpdate:
-				region.update()
+				from treeInterceptorHandler import TreeInterceptor
+				if isinstance(region.obj, TreeInterceptor) and not region.obj.isAlive:
+					log.debugWarning("Skipping region update for died tree interceptor")
+					continue
+				try:
+					region.update()
+				except Exception:
+					log.debugWarning(

```

Comment:
Perhaps this should just be a debug level log if this is an expected path.

```suggestion
					log.debug(
```

Reply:
I'm not sure about this one as I haven't seen it in the wild yet, I just wanted to make sure that a potential error wouldn't cause the update to fail completely. I think I prefer this to stay on debugWarning, we can always reconsider.

Reply:
Makes sense